CHANGELOG.md

```diff
@@ -3,9 +3,22 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## TBD
+## [0.0.8] TBD
+### Added
+* Support for passing certain configurations like Graphite directly to internal CT instance using `--ct-config` argument.
+
+## [0.0.7] 2019-01-28
```

Comment:
For future reference the keepachangelog.com format is `## [1.0.0] - 2017-06-20` (i.e. with a dash separator)

Reply:
Fixed in whole file.